adding prop "phone" to disply filter like on mobile even if you are on desktop (#515)
vtex-apps/search-result: CHANGELOG.md

and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).

## [Unreleased]

> ### Added
> - adding `phone` type on prop `layout` to the `FilterNavigator`.

Reviewer: ```suggestion
### Added
- `phone` option to the `layout` prop  from `FilterNavigator` component.
```
